src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+@import 'variables';
+
+.calendar {
+  margin: 0 auto;
+  width: 706px;
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @each $day, $i in $week {
+    &--start-day-#{$day} :first-child {
```

Comment:
There is an extra space before `:first-child` in the selector `&--start-day-#{$day} :first-child`. This space might cause the selector to not work as intended. It should be `&--start-day-#{$day}:first-child` without the space.